freeze_test.go

```diff
@@ -104,6 +105,55 @@ func TestFreezeErrorFileMissing(t *testing.T) {
 	}
 }
 
+func TestFreezeWindowTitleFilename(t *testing.T) {
+	output := "artichoke-default-title.svg"
+	defer os.Remove(output)
```

Comment:
t.Cleanup should be preferred

Reply:
I was using `defer os.Remove` because of all the tests above. Should them be changed in another PR to clean things up?